tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js

```diff
@@ -84,7 +84,9 @@ boundaryEventData.forEach(({ type, nodeType, eventXMLSnippet }) => {
         .its('xml')
```

Comment:
Might need input on the test. I was able to find the x/y in the code below. But I had no idea how to extract it:

```php
getElementAtPosition(taskPosition).then(getComponentsEmbeddedInShape).then(events => events[0]).then($event => {
  console.log($event);
  console.log($event[0].attributes.transform.textContent; // transform(232,251)
  console.log($event.position()); // no where close to the actual position
});
```

Reply:
`getComponentsEmbeddedInShape` returns an array of jQuery elements, not an event. This is a function we defined and you can see it in `/tests/e2e/support/utils.js`.

You can try running [`getBBox`](https://developer.mozilla.org/en-US/docs/Web/API/SVGGraphicsElement/getBBox) on the underlying SVG elements, instead of `position()`. Then you'd have to convert it to paper coordinates for comparison, which you could use our `getPositionInPaperCoords` util function for.